src/editor/components/modals/file-modal/manage-file-sources/in-progress.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
I think with a more thorough Storybook, we'll find this doesn't generalize to other situations quite as nicely, so I'll probably re-implement this at some point.